signer/src/config/default.toml

```diff
@@ -145,6 +145,27 @@ bootstrap_signatures_required = 2
 # TODO(715): Add default delay (15 seconds? see #701)
 bitcoin_processing_delay = 0
 
+# How many bitcoin blocks back from the chain tip the signer will
+# look for requests.
+# Required: false
+# Environment: SIGNER_SIGNER__CONTEXT_WINDOW
+context_window = 500
+# The maximum duration of a signing round before the coordinator will
+# time out and return an error.
+# Required: false
+# Environment: SIGNER_SIGNER__SIGNER_ROUND_MAX_DURATION
+signer_round_max_duration = 30
+# The maximum duration of a pre-sign request before the coordinator will
```

Comment:
Oh, please add a new line between each parameter and the comments for the next parameter.